importer/tools/export.hs

```diff
@@ -226,7 +232,10 @@ main = do
     repo <- CS.open (argv !! 1)
     let out_path = argv !! 2
     allThings <- expandFileThings $ drop 3 argv
-    let things = map T.pack allThings
+
+    let (match, otherThings) = partition (isPrefixOf "filesystem-") allThings
+    when (length match < 1) needFilesystem
+    let things = map T.pack $ match !! 0 : otherThings
 
```

Comment:
I don't really like this but I can't think of any other, better way to accomplish the same thing.  So, whatever!